src/jit/codegenlegacy.cpp

```diff
@@ -18662,7 +18662,7 @@ regMaskTP CodeGen::genCodeForCall(GenTreeCall* call, bool valUsed)
 #ifdef _TARGET_ARM_
         // Ensure that we don't trash any of these registers if we have to load
         // the helper call target into a register to invoke it.
-        regMaskTP regsUsed;
+        regMaskTP regsUsed = 0;
```

Comment:
@jashook was the windows ARM32 compiler complaining about this? It is strange since there already was another place with exactly the same usage pattern before. Also, the regsUsed is always filled in by the regSet.rsLockReg and it is passed to the regSet.rsUnlockReg only if it was set this way. But obviously, the compiler might not see it.

Reply:
This was caught by the desktop build. The arm failure is unrelated and being fixed by #13932

Reply:
The Mirror is down right now.